<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialogLocate</class>
 <widget class="QDialog" name="QWDialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>行号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxRow"/>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="ckRow">
        <property name="text">
         <string>行增</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>列号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxColumn"/>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="ckColumn">
        <property name="text">
         <string>列增</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>设定文字</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnSetText">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>设定文字</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/title.png</normaloff>:/images/title.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>QWDialogLocate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
